tooling/cli/src/scripts/read-theme-file.worker.ts

```diff
@@ -1,18 +1,51 @@
 import "regenerator-runtime/runtime"
 import path from "path"
+import fs from "fs"
 import { register } from "ts-node"
 import { isObject } from "@chakra-ui/utils"
 import { createThemeTypingsInterface } from "../command/tokens/create-theme-typings-interface"
 import { themeKeyConfiguration } from "../command/tokens/config"
 
+const bold = (text: string) => `\x1b[1m${text}\x1b[22m`
+
+async function importTheme(path: string) {
+  const module = await import(path)
+  const theme = module.default ?? module.theme
+
+  if (!theme)
+    throw new Error(`
+    Theme export not found in module: '${path}'.
+
+    A theme should have a ${bold("default")} export or a ${bold(
+      "theme",
+    )} named export.
+    Found the following exports: ${bold(Object.keys(module).join(", "))}
```

Comment:
NICE! This makes it even easier to identify the problem than only telling them they're missing an export.